core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -177,6 +197,9 @@ var TopicsAndSkillsDashboardPage = function() {
       topicNameField,
       'Create Topic modal takes too long to appear.');
     await topicNameField.sendKeys(topicName);
+    await topicDescriptionField.sendKeys(description);
+    await topicCategoryField.click();
+    await browser.driver.switchTo().activeElement().sendKeys(category + '\n');
```

Comment:
This requires a nested await:
```suggestion
    await (await browser.driver.switchTo().activeElement()).sendKeys(category + '\n');
```
The reason is that `.activeElement()` returns a promise that needs to resolve before we call `.sendKeys`

Reply:
Done.